Monika After Story/game/event-handler.rpy

```diff
@@ -1713,6 +1823,363 @@ init python:
     import store.evhand as evhand
     import datetime
 
+
+    class MASEventContext(mas_utils.IsolatedFlexProp):
```

Comment:
There's on vulnerably . Technically nothing new, but it's less clear here because we obfuscate persistent data via this class.
```py
c = MASEventContext.get()
c.this_is_bad = object()
c.this_is_worse = MyCustomClass()
c.oh_my_god = get_unpickable_data()
```
That can lead to data corruption.

Reply:
okay latest changes should guardrail against bad context data (will update pr text with the new stuff)